src/__test__/utils/cacheRequest.test.js

```diff
@@ -4,42 +4,46 @@ import { /* cacheFetch, */ fetchCachedWork } from '../../utils/cacheRequest';
 enableFetchMocks();
 
 const fakeCacheKeyMappings = {
-  A: 'fd3161a878f67ebf54018720cffd6a66', // pragma: allowlist secret
-  B: '10250a11679234110a1c260d6fd81d3c', // pragma: allowlist secret
-  C: '76bf160685c4c80c67abd9a701da23e6', // pragma: allowlist secret
-  D: '21671038b9ac73c1f08a94c8213cb872', // pragma: allowlist secret
-  E: '33faa711a94a2028b5bae1778126aec0', // pragma: allowlist secret
+  A: '5b995fe9ed8f00cff155a59afa6c523c', // pragma: allowlist secret
+  B: '660bcd9c0cb1fd9ca1d6cde2e2e8a60f', // pragma: allowlist secret
+  C: 'a3324c2021c4855c5c788bffb860b087', // pragma: allowlist secret
+  D: '77b59a783da49797d750d6904f9c5972', // pragma: allowlist secret
+  E: 'ad0f4aeb13f02c99850ab52d473d187a', // pragma: allowlist secret
 };
 
 const fakeData = {
   A: {
     min: 0,
     max: 6.8,
     expression: [0, 0, 0, 1.56, 0, 6.8, 3.4],
+    expressionType: 'raw',
   },
   B: {
     min: 0,
     max: 6.8,
     expression: [0, 0, 0, 2.56, 0, 6.8, 2.56],
+    expressionType: 'raw',
   },
   C: {
     min: 0,
     max: 3.4,
     expression: [0, 0, 0, 3.56, 0, 4.8, 3.4],
+    expressionType: 'raw',
   },
   D: {
     min: 0,
     max: 6.8,
     expression: [0, 0, 0, 1.56, 0, 6.8, 3.4],
+    expressionType: 'raw',
   },
   E: { hello: 'world' },
 };
 
 const fakeCacheContents = {
-  fd3161a878f67ebf54018720cffd6a66: 'A', // pragma: allowlist secret
-  '10250a11679234110a1c260d6fd81d3c': 'B', // pragma: allowlist secret
-  f5c957411a28de68f35e1f5c8a29da7e: 'C', // pragma: allowlist secret
-  '33faa711a94a2028b5bae1778126aec0': 'E', // pragma: allowlist secret
+  '5b995fe9ed8f00cff155a59afa6c523c': 'A', // pragma: allowlist secret
```

Comment:
I wonder how and why this comment `// pragma: allowlist secret` got there

Reply:
It's part of package [`detect-secrets`](https://github.com/Yelp/detect-secrets#inline-allowlisting) to detect secrets in code-bases. Maybe Marcell is using it in the pipeline to sift for sensitive strings?

Reply:
Correct. `detect-secrets` looks for patterns for sensitive credentials (such as AWS, GitHub PATs, crypto keys), and it has an additional function where it looks for any high-entropy string, which I thought was valuable in case we started using something else in the future.

Obviously, hashes fall under this category, so that's why we explicitly ignore them.